editoast/src/views/timetable.rs

```diff
@@ -185,7 +185,7 @@ async fn post(
 async fn delete(
     State(db_pool): State<DbConnectionPoolV2>,
     Extension(auth): AuthenticationExt,
-    timetable_id: Path<TimetableIdParam>,
+    Path(TimetableIdParam { id }): Path<TimetableIdParam>,
```

Comment:
What do you think about keeping `timetable_id` for the rest of the code? (Note: I made only this comment, but this applies to a lot of occurences in the current commit).

```suggestion
    Path(TimetableIdParam { id: timetable_id }): Path<TimetableIdParam>,
```

Reply:
Done in [`6968585` (#9935)](https://github.com/OpenRailAssociation/osrd/pull/9935/commits/6968585670931e0cb03bb27ed147424df3fde1df)